src/main/java/graphql/schema/GraphQLTypeVisitor.java

```diff
@@ -0,0 +1,73 @@
+package graphql.schema;
+
+import graphql.Internal;
+import graphql.util.TraversalControl;
+import graphql.util.TraverserContext;
+
+@Internal
```

Comment:
This should be public: we recently made the general Traverser also public, so nothing prevents us from making this public too